### DISCUSSION TITLE
Regarding scene switching just as a simple timed sequence of pre-created scenes

### DISCUSSION THREAD

**Discussion by DrunkenTyromancer** [WarmUpTill/SceneSwitcher#1036] [Q&A]
[EDIT/UPDATE] ALSO: I see installation instructions for Linux but nothing for WIndows, I can see the link to download the WIndows version, but nothing about how to actually install this in Windows ... it seems to me the first question answered is that perhaps this is not the same plugin as the one that comes with OBS by a similar name that I was looking at, so I might as well try installing this, but I cannot see where I do that, I do not see any executable file in the download, there is no "read me" text file, nothing ... how do I do it?
================================================================================
Perhaps I missed it, but I had a look at the Tools > Automatic Scene Switcher, which I presume is the same thing as this project (?) ... if not, could you please clarify the difference and whether or not this project can do what I want or not, because it seemed like it would from forums but I don't see how when I try to use the thing in OBS ... but I created a progression of scenes, so that when my recording starts, I can do some simple transitions between a 'Start' scene (which is just a welcome slide with some graphical text on it), to an 'Intro' scene, where I slowly fade across to a side-view webcam of the seat I will be sitting in and the USB mic next to it ... then I wanted to transition from that to a scene 'Main' with a front view webcam, and then I have a variation of that scene where either a Discord videocall or a Signal video call can sit in the top right corner as a box containing head and shoulders of someone I am talking to ... I can always get out of my chair to manually trigger this last switch, and any additional scenes I may make, for example if I have multiple guests to talk to and I want to have the screen in a grid of boxes ... but the main issue is I want the transition from start > intro > main to just happen on a timer, either beginning with a manual trigger, or being triggered by "start recording" or "start streaming"

BUT ... when I went to use that menu item Tools > Automatic Scene Switcher, it brings up a window with 2 drop down menus at the top, neither of which has a label telling you what it is, and only the RHS is obvious as it contains the names of the scenes, but the LHS just has what I presume are a bunch of windows currently open ... but I don't care about any of these, I just want it to create a timed transition between 3 scenes that are all ready to go, as are all the windows I need for them to function, and some of those windows are on an external monitor and configured in such a manner as that I do not have to select them or anything, they are always in focus for recording.

Can this be done, is it part of this menu item I am looking at, are the two scene switchers the same, or is this one here in GitHub different from that one in the menu ... what's the story? I tried to search for a similar discussion, but I am not on GitHub very often, so I am not sure if there are just no other discussions about this or I am searching in the wrong way, and I am mentally exhausted and sleep deprived from trying to get a RODECaster Duo set up and configured where nothing seems to be explained properly unambiguously and so on ... so perhaps I am missing something obvious, that is entirely possible.

**Comment by nacalai** [comment]
Automatic Scene Switcher (that comes with OBS) is not the same as Advanced Scene Switcher. But I can understand the confusion.

Download and install from here https://github.com/WarmUpTill/SceneSwitcher/releases (under "Assets") and you'll see it in Tools - Advanced Scene Switcher and you can create the sequence of macros that you are after :)

**Comment by DrunkenTyromancer** [comment]
I do not see it under Tools because I do not know which folder to unzip the files into, and whether or not to include the parent folder and so forth ... OBS can't just see it and use it anywhere in my file system, especially not without telling it where it is, therefore it must have to go to a particular place to be detected, but that is what I cannot see anywhere in documentation, where it is I am supposed to put it, and when you unzip with 7zip for example, do I use the function to prevent a duplication of the root folder, and which folder do I extract it all into ...

**Comment by WarmUpTill** [comment]
I have attached a small video which shows how to download and install the plugin for windows.
I hope that helps.

![Example](https://github.com/WarmUpTill/SceneSwitcher/assets/19472752/39b6a4e0-d720-451f-acfa-ef8630ed234e)

If you can't get it to work feel free to contact me directly on the OBS forums via PM and we can set up a quick call where I walk you through all of the setps.

**Reply by therentabrain** [reply]
Because seriously @WarmUpTill is the best
